docs/coding-standards/tests/common-rule/mock-file-placement.md

```
@@ -0,0 +1,67 @@
+     - 新しいモックファイルを作成してはならない（再エクスポート用ラッパーも禁止）
+     - テストコードで共有モックのパスを直接指定すること
+   - 見つからなかった場合 → モック対象クラスのtestsディレクトリに新規作成
+
+3. **共有モックの配置先**
+   - ポート/インターフェースレベルのモック → `tests/unit/{layer}/ports/{category}/{InterfaceName}/mocks/`
+   - 例：`ITabsGateway` → `tests/unit/application-business-rules/ports/gateway/ITabsGateway/mocks/`
+   - 例：`IRewriteRuleRepository` → `tests/unit/application-business-rules/ports/gateway/IRewriteRuleRepository/mocks/`
+
+**重要**: この手順を省略すると、PRレビューで重複モックの指摘を受けます。
+
+#### 既存コードへの適用
```

markdownlint-cli2 (0.20.0)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)